core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/appRoutesModels.scala

```diff
@@ -27,7 +27,9 @@ final case class CreateAppRequest(kubernetesRuntimeConfig: Option[KubernetesRunt
                                   appType: AppType,
                                   diskConfig: Option[PersistentDiskRequest],
                                   labels: LabelMap = Map.empty,
-                                  customEnvironmentVariables: Map[String, String])
+                                  customEnvironmentVariables: Map[String, String],
+                                  descriptorPath: Option[String],
```

Comment:
if this path is a uri, can we use a `org.http4s.Uri` for its type

Reply:
Yeah +1.. for now we are supporting Uris, in the future might allow things like Dockstore references but that's TBD.